### ISSUE TITLE
Clarify what variables can be used in Connector Result expressions

### ISSUE THREAD

**Issue by nloding** [camunda/camunda-docs#3555]
Process variables cannot be used in the "Result expression" field in Connectors, because that expression is evaluated _inside_ the Connector Runtime, which does not have access to data unless it was specifically provided. I know this is true of the REST connector, and looking at the Connector code, it looks to be true of the other Connectors as well. It would be good to confirm with the Connectors team, of course.

Here is a forum thread with relevant information: https://forum.camunda.io/t/camunda-8-loop-through-response-body/51489/5

**Comment by akeller** [comment]
@sbuettner can you have a look at this docs issue?

**Comment by sbuettner** [comment]
@nloding That is correct. The Connector only receives access to the variables specified in the annotation and the result expression is evaluated only against the Connector result. @nloding Where would you expect this kind of information in the docs?

**Comment by akeller** [comment]
@nloding any more feedback on this? 

I could see a brief mention in this section: https://docs.camunda.io/docs/components/connectors/use-connectors/#result-expression

**Comment by nloding** [comment]
Not sure why I didn't reply last year ....... oops. There is a statement in the docs that says:

> To ensure process isolation, note that Connectors do not have access to process instance variables.

I think it could be further clarified with something like: `The only variables the result expression has access to is data returned from the connector" but that's not completely accurate (the word "returned" is misleading here).

**Comment by sbuettner** [comment]
@nloding Would be nice if you could create a PR with your proposal.
